### PR TITLE
chore: update to rules_js 1.1.0

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ jobs:
         runs-on: ubuntu-latest
 
         strategy:
+            fail-fast: false
```
Comment (jbedard) [inline .github/workflows/ci.yaml:20]: Are the changes in this file necessary?

File: e2e/worker/WORKSPACE [hunk 2]
```diff
@@ -22,9 +22,10 @@ load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
 
 npm_translate_lock(
     name = "npm",
+    lifecycle_hooks_no_sandbox = False,
```
Comment (jbedard) [inline e2e/worker/WORKSPACE:25]: Why was this required?

**Review by jbedard — APPROVED** [review]

File: e2e/worker/WORKSPACE (hunk 2 above)
Comment (gregmagolan) [reply e2e/worker/WORKSPACE:25]: It was. One of the @nestjs lifecycle hooks fails outside of the sandbox. Didn't look further into why. I'll comment there.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (gregmagolan) [reply .github/workflows/ci.yaml:20]: bzlmod was failing on CI; we should add fast-fail throughout the repos actually. I'll prefactor it.
